Sources/Plasma/PubUtilLib/plDrawable/plAccessSpan.h

```diff
@@ -139,5 +142,23 @@ inline plAccessVtxSpan& plAccessSpan::AccessVtx()
 }
 
 
+const plAccessVtxSpan& plAccessSpan::AccessVtx() const
+{
+    switch (fType)
+    {
+    case kTri:
+        return fAccess.fAccessTri;
+    case kParty:
+        return fAccess.fAccessParty;
+    case kVtx:
+        return fAccess.fAccessVtx;
+
+    case kUndefined:
+    default:
```

Comment:
```suggestion
    case kUndefined:
    [[fallthrough]]
    default:
```

Reply:
`[[fallthrough]]` is not needed for completely empty case statements (esp. since it's common to group several case statements together as a sort of code lookup table)